### ISSUE TITLE
Upgrading from 4.0.0 to 4.0.1 or 4.1.0 causes many false positives of `no-quoteless-attributes`

### ISSUE THREAD

**Issue by jacobq** [ember-template-lint/ember-template-recast#221]
Reproduction repository: https://github.com/jacobq/ember-template-recast-bug-repro

1. `git clone git@github.com:jacobq/ember-template-recast-bug-repro.git`
2. `cd ember-template-recast-bug-repro`
3. `git checkout b58ee835eecd07524d40a25ddb4c83e5876ea55e`
4. `npm install` or `yarn`
5. `npm run lint:hbs`

```
> junk-app@0.0.0 lint:hbs /home/jquant/junk-app
> ember-template-lint .

/home/jquant/junk-app/app/templates/application.hbs
  1:12  error  Attribute class should be either quoted or wrapped in mustaches  no-quoteless-attributes

✖ 1 problems (1 errors, 0 warnings)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! junk-app@0.0.0 lint:hbs: `ember-template-lint .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the junk-app@0.0.0 lint:hbs script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2020-02-21T17_36_02_251Z-debug.log
```

Adding `"**/ember-template-recast": "4.0.0"` to package.json's `resolutions` section (using yarn) fixes the problem.

**Comment by jacobq** [comment]
The problem appears to have started happening between v4.0.0 and v4.0.1

**Comment by rwjblue** [comment]
Ya, @jacobq I fixed ember-template-lint over in https://github.com/ember-template-lint/ember-template-lint/pull/1123. I'll get that released in another beta so folks aren't hitting the errors you were.

Sorry about the troubles!

**Comment by rwjblue** [comment]
The fix has been released in ember-cli-template-lint@2.0.2.
